packages/wrangler/src/publish.ts

```diff
@@ -88,6 +88,12 @@ export default async function publish(props: Props): Promise<void> {
     file = path.resolve(path.dirname(__path__), build.upload.main);
   }
 
+  if (config.site?.["entry-point"]) {
```

Comment:
could you add this for the dev command as well please?

Reply:
Added.